### DISCUSSION TITLE
Array of Paragraphs

### DISCUSSION THREAD

**Discussion by megaszczyk** [dolanmiu/docx#1138] [Q&A] [answered]
Hi, I'm using JavaScript to create an array of paragraphs following the example of cv creation.
After creating the document I can't open it.
Any help would be appreciated.

**Comment by dolanmiu** [comment, marked as answer]
Probably a syntax error
Double check what you wrote is correct
Try use TypeScript for compile time checking

**Reply by megaszczyk** [reply]
Hi, thank you for your response.
I have implemented this part on its own, and it works absolutely fine.
I'll be looking through the code and try to identify what is causing the problem.

**Reply by megaszczyk** [reply]
I have figured out that adding a section type solved the problem.
